<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="INBLRNB0299" name="testngsnippet.TestNgAnnotations" tests="6" failures="0" timestamp="10 Nov 2019 06:32:45 GMT" time="1.034" errors="1">
  <testcase name="f1" time="0.006" classname="testngsnippet.TestNgAnnotations"/>
  <testcase name="f3" time="0.001" classname="testngsnippet.TestNgAnnotations"/>
  <testcase name="f2" time="0.003" classname="testngsnippet.TestNgAnnotations"/>
  <testcase name="f5" time="0.014" classname="testngsnippet.TestNgAnnotations"/>
  <testcase name="f6" time="1.010" classname="testngsnippet.TestNgAnnotations">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method testngsnippet.TestNgAnnotations.f6() didn&#039;t finish within the time-out 1000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method testngsnippet.TestNgAnnotations.f6() didn't finish within the time-out 1000
at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:384)
at java.util.concurrent.FutureTask.setException(FutureTask.java:251)
at java.util.concurrent.FutureTask.run(FutureTask.java:271)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- f6 -->
  <testcase name="f4" classname="testngsnippet.TestNgAnnotations">
    <skipped/>
  </testcase> <!-- f4 -->
</testsuite> <!-- testngsnippet.TestNgAnnotations -->
